### PR TITLE
Feature/broadcast cards

### PR DESCRIPTION
**Pull request by jokerengine**

### CODE REVIEW

File: server/src/main/java/jass/server/util/CardUtil.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import jass.lib.servicelocator.Service;
+
+/**
+ * Creates a game once 4 clients are searching for a game in the lobby.
+ *
+ * @author Thomas Weber
```
Comment (tjw52) [inline server/src/main/java/jass/server/util/CardUtil.java]: @author should be you :p

File: server/src/main/java/jass/server/util/CardUtil.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+public final class CardUtil implements Service {
+    /**
+     * The service name.
+     */
+    public static final String SERVICE_NAME = "CardUtil";
+
+    /**
+     * The logger to print to console and save in a .log file.
+     */
+    private static final Logger logger = LogManager.getLogger(SearchGameUtil.class);
```
Comment (tjw52) [inline server/src/main/java/jass/server/util/CardUtil.java]: Change logger to CardUtil

File: server/src/main/java/jass/server/util/CardUtil.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+     *
+     * @param client The client to add.
+     */
+    public void addClientToSearchGame(final ClientUtil client) {
+        /*if (!clients.contains(client)) {
+            clients.add(client);
+            logger.info("Added new client " + client.getUsername() + " to waiting list. Total of " + clients.size() + " users searching for a game");
+            this.createNewGame();
+        }*/
+    }
```
Comment (tjw52) [inline server/src/main/java/jass/server/util/CardUtil.java]: Can be deleted

**Review by tjw52 — COMMENTED** [review]
Nice job boi :D I didn't really see anything to change or find any bugs, 3 really minor things in CardUtil

Are you coordinating with Sasa on making sure that the broadcast of cards is registered to the GUI? I guess that will be in her PR then?

In RoundEntity we will have to add a property for the chosen game mode but as it's not really part of your task I didn't comment it in...

File: server/src/main/java/jass/server/util/CardUtil.java (hunk 2 above)
Comment (jokerengine) [reply server/src/main/java/jass/server/util/CardUtil.java]: ah right..thanks 👍

File: server/src/main/java/jass/server/util/CardUtil.java (hunk 1 above)
Comment (jokerengine) [reply server/src/main/java/jass/server/util/CardUtil.java]: haha, oops

**Comment by jokerengine** [comment]
thanks thanks :D 
yes, I'm coordinating with Sasa. I've merged my branch into her branch today and did some work with her too. The cards are now being displayed, but she needs to make it look pretty

**Pull request merged**